app.py

```diff
@@ -46,14 +49,33 @@ def right():
 @app.route("/automate_vehicle")
 def automate_vehicle():
     # TODO:: Instantiate the class and call required functions
-    initPoint = Coordinates('basic_science',(12.96626,77.71211))
-    finalPoint = Coordinates('ganesha_statue',(12.96598,77.71148))
-    automate_drive = Automation(initPoint,finalPoint)
+    initPoint = Coordinates('basic_science', (12.96626, 77.71211))
+    finalPoint = Coordinates('ganesha_statue', (12.96598, 77.71148))
+    automate_drive = Automation(initPoint, finalPoint)
     automate_drive.automate_linear()
     return "a"
 
 
+@app.route("/init_tree")
+def init_tree():
+    global gt
+    gt = GeocodeTree()
+    return "a"
+
+
+@app.route("/add_tree")
+def tree():
+    gt.add_tree([12.96691, 77.71112])
+    return "a"
+
+
+@app.route("/save")
+def save():
+    gt.save_and_return()
+    return "a"
```

Comment:
Route to save the graph object with new tree nodes.
More explanation in comment relating to geocodeTree.py

Reply:
Why a separate route?

Reply:
For each and every tree saving is computationally not worth it.
Plus in future this could open up space for undoing the function of "Add Tree", "Remove Tree"

So currently its more streamlined(s/w).
you add all trees. then click on save .

Reply:
And even look at the different applications of putting this up on a GUI
User could enter a separate view where the tree names are displayed , with other nodes.,
We could have him edit over there.
Then in the end just click on "Save"